### PR TITLE
more clear learning rate config

### PR DESCRIPTION
**Pull request by jeremyjordan**
## What does this PR do?

We were only using the model `learning_rate` attribute to ensure that it was logged, but now that we're logging all of the optimizer config we should only specify the learning rate in the optimizer config.

### CODE REVIEW

**Review by huffmanjohnf — APPROVED** [review]
lgtm. Great idea to move it to the optimizer.
